GameServerLib/Packets/PacketHandlers/HandleMove.cs

```diff
@@ -66,54 +66,55 @@ private List<Vector2> ReadWaypoints(byte[] buffer, int coordCount, IMap map)
                 coordCount++;
             }
 
-            var mapSize = map.NavGrid.GetSize();
```

Comment:
Your PR still uses the redundant method `GetSize`. This method returns the `Center Point` of the Map not its `Size`.
This means you should use the `MiddleOfMap` property of the Navigation Grid instead of that function.

Reply:
I'm talking about the solution only. LeaguePackets should be integrated to delete this function. Basically, LeaguePackets already doing the correct reading of Waypoints and the current code doesn't use LeaguePackets for reading the MoveRequest.

So cleaning this function is not recommended (as it would be deleted).

I don't have any problem with optimizations to that function, but in the future this function will be deleted.

Reply:
Well we can clean it up for now and you can deal with it later with your PR.
This PR does not intend to change any logic of Packets or such. It only aims to clean up and add new things to the Navigation Grid. Which in result can change certain code in other places such as LeaguePackets